ui/admin/tests/acceptance/onboarding/selectors.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export const DO_THIS_LATER_BTN = '.rose-form-actions [type="button"]';
+export const COPY_BTN_TEXT = '.hds-copy-snippet__text';
```

Comment:
I will suggest this to be renamed: COPY_TEXT_BTN to follow the current pattern, [in example here](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/helpers/selectors.js#L6-L8)